includes/discovery/ports.inc.php

```diff
@@ -94,6 +94,10 @@
     if (is_port_valid($snmp_data, $device)) {
         port_fill_missing_and_trim($snmp_data, $device);
 
+        if ($device['os'] == 'vmware-vcsa' && preg_match('/Device ([a-z0-9]+) at .*/', $snmp_data['ifDescr'], $matches)) {
+            $snmp_data['ifDescr'] = $matches[1];
```

Comment:
Shouldn't this be ifName?

Reply:
Sure, can be, just needs an extra change.